src/content/en/updates/2017/09/sensors-for-the-web.md

```diff
@@ -52,17 +53,18 @@ number of advantages:
 - Generic Sensor API is a framework, i.e. it can be easily extended
   with new sensor classes and each of these classes will keep the generic
   interface. The client code initially written for one sensor type can be
-  reused for another with very little modifications!
+  reused for another with very few modifications!
 - You can configure the desired sensor data. For example, request sensor reading
-  delivery rate suitable for your application.
-- You can detect whether sensor is available on the platform.
+  delivery rate suitable for your application needs.
+- You can detect whether a sensor is available on the platform.
 - Sensor readings have high precision timestamps, enabling better
-  synchronization with other activities in your applicaion.
+  synchronization with other activities in your application.
 - Sensor data models and coordinate systems are clearly defined, allowing
   browser vendors to implement interoperable solutions.
-- The Generic Sensor based interfaces are not bound to DOM (neither navigator
-  nor window objects), and it opens up new opportunities of using same API from
-  service workers or implementing Generic Sensor API in headless JS runtimes.
+- The Generic Sensor based interfaces are not bound to the DOM (neither
+  navigator nor window objects), and it opens up future opportunities of using
```

Comment:
the object is called Navigator and Window but are exposed via the navigator accessor/namespace or whatnot... maybe we need to rewrite that